package.json

```diff
@@ -1,17 +1,30 @@
 {
-  "author": {
-    "name": "Erik Rose",
-    "url": "https://grinchcentral.com"
+  "name": "fathom",
```

Comment:
I think `name` is a required field in package.json, but not sure if we want to go w/ "fathom", since it's already used in npm.
Or maybe we get fancy and give it a [scoped name](https://docs.npmjs.com/misc/scope), like `@mozilla/fathom` to avoid collisions.